### ISSUE TITLE
IMF Fix

### ISSUE THREAD

**Issue by NLangeweg** [Cold-War-Project/CWP-Main#629]
I already have a mod for this but that one has some small issues. The problem was that the contribution of every country was also the reserve pool of the IMF.

So lets say the US dumped 90 million and Germany 10 million IRM then Germany would have 10% contribution. When someone then takes a loan of lets say 50 million it will keep the percentages the same but the values decreases. So now the IMF contribution of the US would be 45 million and germany 5 million.

This creates problems if lets say Austrialia takes a loan consisting of the entire IMF reserve. Because then everyone's contribution would be decreased to 0.

And then if Bolivia decides to be the first to contribute 1.000 IMR to the emptied reserves they would have near a 100% contribution, receiving the entire paycheck from the debt payments which it will use to put back into the IMF and becoming an economic super power. If the country has -99% currency value this will give that country millions of extra income aswell.

tldr: need to add actual reserves to IMF.
